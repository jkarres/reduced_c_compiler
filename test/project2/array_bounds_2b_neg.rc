// reading from an invalid location (negative)

int[9] ia;

function : void main() {

  int i;

  for (i = 8; i > -2; --i) {
    cout << ia[i] << endl;
  }
}
