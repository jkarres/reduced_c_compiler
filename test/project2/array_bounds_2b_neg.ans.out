0
0
0
0
0
0
0
0
0
Index value of -1 is outside legal range [0,9).
